Fix icons of map preview of auto lobby being miss placed (#6768)
FAForever/fa: changelog/snippets/fix.6768.md

> - (#6768) Fix icons of the map preview of auto being miss placed
> 
> This was the case for all users that use the in-game UI scaling.

Reviewer: ```suggestion
- (#6768) Fix icons in the map preview of the autolobby being misplaced when using UI scaling.

```
